docs/configuration.asciidoc

```diff
@@ -112,10 +110,10 @@ or a negative value, timeouts will be disabled.
 | `ELASTIC_APM_SECRET_TOKEN` |         | "A random string"
 |============
 
-This string is used to ensure that only your agents can send data to your APM server.
+This string is used to ensure that only your {agent}s can send data to your APM server.
```

Comment:
Does the s need to go in the brackets or outside?

Reply:
Outside. Thanks.